v0/destinations/facebook_pixel/transform.js

```diff
@@ -105,16 +105,21 @@ const handleOrder = (message, categoryToContent) => {
       const pId =
         products[i].product_id || products[i].sku || products[i].id || "";
       contentIds.push(pId);
+      // required field for content
+      // ref: https://developers.facebook.com/docs/meta-pixel/reference#object-properties
       const content = {
         id: pId,
-        quantity: products[i].quantity,
-        item_price: products[i].price
+        quantity: products[i].quantity || message.properties.quantity || 1,
+        item_price: products[i].price || message.properties.price
```

Comment:
Is it the right way to take price? Let's say some customers put the total price in properties.price by taking sum of individual prices of each products. Now if the price of one product is zero then it will get the total price.

Reply:
The ideal way will be to put price for individual products, which is according to our spec. `message.properties.price` is kept to ensure backward compatibility with device mode.